### DISCUSSION TITLE
I would like to know how to enable these features using a recipe

### DISCUSSION THREAD

**Discussion by Rodien** [OrchardCMS/OrchardCore#15586] [Q&A] [answered]
Hi guys,

I would like to know how I can enable the following features: User Password Reset and Two-Factor Authentication,  Email.

Additionally, when I enable User Password Reset, how can I automatically enable the settings found under Admin Dashboard > Security > User Password Reset?

![image](https://github.com/OrchardCMS/OrchardCore/assets/50739829/a717933b-a2fc-47e9-85cc-15511e69c803)

**Comment by Piedone** [comment, marked as answer]
[This video](https://www.youtube.com/watch?v=2c5pbXuJJb0) explains the basics of the Deployment module as a start. In this particular case, you can export what you need with a Deployment Plan that has the "All Features" (for enabling features from a recipe) and the "Reset Password settings" (for the two checkboxes) steps. In the recipe, you'll see how these work.

I encourage you to try to add steps for other settings, and see what recipe is produced. You'll notice that it follows a simple pattern, and you can easily edit by hand too :).

**Reply by Rodien** [reply]
Hey @Piedone, thank you for the response. I'm going to take a look at the video.
